imports/styles/app.css

```diff
@@ -56,9 +39,10 @@ body {
 .reveal {
   width: fit-content;
 }
-.button.small {
-  margin-bottom: 0rem;
+.fa-trash-alt:hover {
```

Comment:
it's not good practice to style direcly the class that defines the icon. if tomorrow I tell you to change it to `fa-trash`or `fa-times` it will not work anymore. if you want to apply styling to your icons, use your own class, like that you have controll.